### ISSUE TITLE
Needs a sample repo

### ISSUE THREAD

**Issue by isidore** [texttest/dbtext#2]
something that shows the files and structure

**Comment by emilybache** [comment]
You're right, this would be great. At the moment all there is is this repo:

https://github.com/emilybache/CustomerSync-Refactoring-Kata/tree/with_tests

the 'with_tests' python version has example usage.
